test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup/tasks/main.yml

```diff
@@ -0,0 +1,128 @@
+---
+## Task file for setup/teardown AWS resources for aws_ssm integration testing
+- block:
+    - name: set up aws connection info
+      set_fact:
+        aws_connection_info: &aws_connection_info
+          aws_access_key: "{{aws_access_key}}"
+          aws_secret_key: "{{aws_secret_key}}"
+          security_token: "{{security_token}}"
+          region: "{{aws_region}}"
+      no_log: yes
+
+    - name: Install Session Manager Plugin for Debian/Ubuntu
+      include_tasks: debian.yml
+      when: ansible_distribution == "Ubuntu" or ansible_distribution == "Debian"
+
+    - name: Install Session Manager Plugin for RedHat/Amazon
+      include_tasks: redhat.yml
+      when: ansible_distribution == "CentOS" or ansible_distribution == "RedHat" or ansible_distribution == "Amazon"
+
+    - name: Install Boto3
```

Comment:
Typically Ansible tests install things from pip in a virtual environment so the changes can be cleaned up at the end of the test. It also allows non-privileged users to run the tests (since just installing with pip install <package> may fail, indicating that the --user flag needs to be used).

Reply:
`needs/root` was added:
https://github.com/ansible/ansible/blob/c84d89a2cf3f1bdb67dc6cbb7c511348f366ce03/test/integration/targets/connection_aws_ssm/aliases#L5